bin/tessel-2.js

```diff
@@ -116,7 +121,7 @@ parser.command('crash-reporter')
     flag: true,
     help: 'Test the Crash Reporter.'
   })
-  .help('Crash Reporter enable/disable');
+  .help('See Crash Reporter status');
```

Comment:
I've updated it to use the same language that `t2 wifi` help uses: `Configure the Crash Reporter.`